modules/olm-enabling-operator-for-multi-arch.adoc

```diff
@@ -0,0 +1,85 @@
+// Module included in the following assemblies:
+//
+// * operators/operator_sdk/osdk-generating-csvs.adoc
+
+[id="olm-enabling-operator-for-multi-arch_{context}"]
+= Enabling your Operator for multiple architectures and operating systems
+
+As an Operator author, you can specify whether your Operator supports multiple
+architectures. If your Operator supports variants other than AMD64 and
+Linux, your CSV must list the supported variants for the Operator to run
+properly.
+
+[IMPORTANT]
+====
+To support listing multiple architectures and operating systems, your Operator
+image referenced in the CSV must be a manifest list image. The image should also
+be specified using a digest (SHA) and not by a tag for it to work properly in
+restricted network, or disconnected, environments.
```

Comment:
@shawn-hurley @emmajiafan Thanks for review. Update the note here to also mention referencing by SHA.

Reply:
I've actually moved these bits into a Prerequisites list before the Procedure to be more obvious.